src/index.js

```diff
@@ -97,6 +99,8 @@ export default class JSONTree extends React.Component {
         <JSONNode
           {...{ postprocessValue, hideRoot, styling, ...rest }}
           keyPath={hideRoot ? [] : keyPath}
+          isRootTypeOfNodeIsArray={objType(value) === 'Array'}
```

Comment:
Don't do this here. We should check this in our code, and set maxClickableNodeDepth.

remove isRootTypeOfNodeIsArray